package/boot/arm-trusted-firmware-mediatek/Makefile

```diff
@@ -174,6 +174,15 @@ define Trusted-Firmware-A/mt7986-spim-nand-ddr4
   NAND_TYPE:=spim:2k+64
 endef
 
+define Trusted-Firmware-A/mt7986-spim-nand-ddr4-4k
```

Comment:
`mt7986-spim-nand-4k-ddr4` as a name would make more sense to me, because the `4k` apply to `spim-nand` and not to `ddr4`